README.mediawiki

```diff
@@ -49,6 +49,14 @@ If you are also using Zenoss to monitor the guest operating system running withi
 
 == Changes ==
 
+;1.2.4
+* zOpenStackInsecure option was added to skip SSL verification. ([https://jira.zenoss.com/browse/ZEN-23460 ZEN-23460])
```

Comment:
You can remove the URL part of the tickets here.
* Also you should explain here or above that this fix/option allows self-signed  or third-party certificates that are not verifiable.